cmd/keytransparency-server/Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN go get -tags="mysql" ./cmd/keytransparency-server
 ENTRYPOINT /go/bin/keytransparency-server \
 	--addr="$HOST:$RPC_PORT" \
 	--db="${DB_USER}:${DB_PASSWORD}@tcp(${DB_HOST})/${DB_DATABASE}" \
-	--vrf="$VRF_KEY_PATH" \
+	--vrf="$VRF_PRIV" \
```

Comment:
Is this change a mistake? I don't see a corresponding VRF_PRIV ENV variable above